common/viewer.h

```diff
@@ -190,7 +192,7 @@ namespace rs2
 
         press_button_model grid_object_button{ u8"\uf1cb", u8"\uf1cb",  "Configure Grid", "Configure Grid", false };
 
-        viewer_model(context &ctx_);
+        viewer_model(context &ctx_, bool disable_log_to_console = false );
```

Comment:
For now this is OK, but if there will be more options controlled then we need to pass in a more flexible way, like JSON.